options/locale/locale_en-US.ini

```
@@ -300,6 +300,8 @@ authorization_failed = Authorization failed
 authorization_failed_desc = The authorization failed because we detected an invalid request. Please contact the maintainer of the app you've tried to authorize.
 disable_forgot_password_mail = Account recovery is disabled. Please contact your site administrator.
 sspi_auth_failed = SSPI authentication failed
+password_pwned = Password has been pwned too many times
```

```suggestion
password_pwned = The password you chose is on a list of stolen passwords previously exposed in public data breaches. Please try again with a different password.
```
Maybe this translation could also link to https://haveibeenpwned.com/Passwords somewhere like:

The password you chose is on a [list of stolen passwords](https://haveibeenpwned.com/Passwords) previously exposed in public data breaches. Please try again with a different password. Not a fan of HTML in translations but something could be helpful perhaps